src/core/utils/ping_sender.hpp

```diff
@@ -65,6 +66,25 @@ class PingSender : public InstanceLocator, private NonCopyable
      */
     typedef otPingSenderReply Reply;
 
+    /**
+     * This class represents the statistics of several ping requests.
+     *
+     */
+    struct Statistics : public otPingSenderStatistics
+    {
+        Statistics() { this->Clear(); }
+
+        void Clear()
+        {
+            mTotalPingCount     = 0;
+            mSentCount          = 0;
+            mReceivedCount      = 0;
+            mTotalRoundTripTime = 0;
+            mMinRoundTripTime   = ot::NumericLimits<uint32_t>::Max();
+            mMaxRoundTripTime   = ot::NumericLimits<uint32_t>::Min();
```

Comment:
We can skip the `ot::` since the `PingSender` itself is already under namespace `ot`.
```suggestion
            mMinRoundTripTime   = NumericLimits<uint32_t>::Max();
            mMaxRoundTripTime   = NumericLimits<uint32_t>::Min();
```